docs/docsite/rst/playbooks_variables.rst

```diff
@@ -884,6 +884,8 @@ Basically, anything that goes into "role defaults" (the defaults folder inside t
 
 .. [1] Tasks in each role will see their own role's defaults. Tasks defined outside of a role will see the last role's defaults.
 .. [2] Variables defined in inventory file or provided by dynamic inventory.
+.. [3] When created with set_facts's cacheable option, variables will have the high precedence in the play,
```

Comment:
Should this be multiple levels?

* host facts
* variables created with cacheable set_facts in a previous play when read from the cache
<..>
*  variables created with set_facts in current play

Is there anything else to distinguish the precendence of host_facts  and 'cached set_facts'?

Reply:
no, they update the same dict, it is order in which they were set, but not by their nature